### ISSUE TITLE
Add NetBSD 32bit and 64bit images to the netboot.xyz

### ISSUE THREAD

**Issue by novaTopFlex** [netbootxyz/netboot.xyz#1437]
**Describe the solution you'd like**

I would like NetBSD images to be added to the BSD menu on netboot.xyz. NetBSD is another (relatively) major distribution "distro" of the BSDs and I believe that since NetBSD is an appropriate basis for routers and the like, NetBSD deserves to be properly added to the netboot.xyz menu.

**Describe alternatives you've considered**

If the 32-bit NetBSD image is not available, I would still like to include the 64-bit version in the appropriate context (x86_64).

**Additional context**

Right now, the only two (2) BSD systems available from netboot.xyz are OpenBSD and FreeBSD, but I believe that NetBSD may actually be slightly more commonly used in certain environments and niches and am not sure how exactly to add such NetBSD images to netboot.xyz.

**Comment by antonym** [comment]
https://github.com/netbootxyz/netboot.xyz/issues/78

Not currently any good paths to use NetBSD with iPXE. If that changes or someone knows of a way to load them, please update the issue.

**Comment by LuboGD** [comment]
Does NetBSD currently work by loading its ISO on a netboot.xyz USB flash drive? It is not included in the list I found (probably the same as the previous poster): https://netboot.xyz/docs/faq/

I ran across another multiboot USB project, and their grub configs for NetBSD worked on the first try, so they seemed good. Maybe incorporate some of that code and expand netboot.xyz's list of OSes, at least with regards to booting ISO from USB via grub or memdisk?

URL of other project with NetBSD entries:
https://github.com/aguslr/multibootusb/blob/main/mbusb.d/netbsd.d/generic.cfg

Also, if possible, it would be great if you looked at adding Adelie and antiX. Both Linux, both make live disks, neither is in the FAQ list.

Thank you for your consideration.

Yes, I do dabble with older hardware.
